Lib/test/test_fractions.py

```diff
@@ -1633,6 +1633,10 @@ def test_complex_handling(self):
                                  message % ("divmod()", "complex", "Fraction"),
                                  divmod, b, a)
 
+    def test_three_argument_pow(self):
+        self.assertTypedEquals(pow(F(3), 4, 5), F(1))
```

Comment:
Fix this to test exception type and message.

Reply:
done